tests/test_apig_wsgi.py

```diff
@@ -39,7 +39,7 @@ def make_event(
     qs_params=None,
     qs_params_multi=True,
     headers=None,
-    headers_multi=True,
+    headers_multi=False,
```

Comment:
Can you not flip this please, and instead add some tests explicitly setting it to `False`? It seems it's on for API Gateway, so I'd rather have most of the tests come through that "default". I know it will mean updating a bunch of the tests.

Reply:
Fair point, I'll clean that up :)